src/app/routes/utils/fetchPageData/utils/getUrl/index.js

```diff
@@ -0,0 +1,17 @@
+import { getQueryString, getUrlPath } from '#lib/utilities/urlParser';
+import getBaseUrl from '../getBaseUrl';
+import onClient from '#lib/utilities/onClient';
+import isLive from '#lib/utilities/isLive';
+
+const baseUrl = onClient()
+  ? getBaseUrl(window.location.origin)
+  : process.env.SIMORGH_BASE_URL;
+
+export default pathname => {
+  if (!pathname) return '';
+
+  const ampRegex = /\.amp$/;
```

Comment:
Note, this is the fix for the regex

Reply:
I was wondering just now if we should have a standard file having generic regexes in simorgh as a standard because there might be an instance where a similar amp check will be totally different (and seem buggy). Just a personal thought though. [see similar fix](https://github.com/bbc/simorgh/pull/8032)

Reply:
Good shout @rhenshaw56 

Just found a few duplicate amp regexes:
https://github.com/bbc/simorgh/blob/latest/src/app/routes/utils/fetchPageData/utils/getRouteProps/routeFallbackParams/index.js#L6
https://github.com/bbc/simorgh/blob/latest/src/app/routes/utils/isAmpPath/index.js#L5
https://github.com/bbc/simorgh/blob/latest/src/app/contexts/RequestContext/getMetaUrls/index.js#L4

I think it would be good to put the regex in this PR into a const file somewhere use it in these places?